src/core/Refiner.ml

```diff
@@ -1170,7 +1170,7 @@ struct
       RM.lift_cmp @@
       Sem.con_to_cof @@
       D.CofBuilder.meet @@
-      List.map (D.CofBuilder.eq D.Dim1) dims
+      List.map D.CofBuilder.eq1 dims
```

Comment:
@favonia This change seems to have made a change in behavior, but that sounds impossible to me! Can you understand it? Look at the `test.expected`